packages/mjml-section/src/index.js

```diff
@@ -302,7 +302,7 @@ export default class MjSection extends BodyComponent {
         ${this.htmlAttributes({
           align: 'center',
           class: this.getAttribute('css-class'),
-          background: this.getAttribute('background-url'),
+          background: this.getAttribute('background-url'), // TODO: <table background="..." is not valid and we need to put image bacground there through style, see https://developer.mozilla.org/cs/docs/Web/HTML/Element/table
```

Comment:
`background` is an old & valid property on `table` element. Some clients like Yahoo doesn't support the `background` property properly and `background` act as a fallback. So please remove both comments.

Reply:
Let's tell this in comments then - `// FYI: background is a deprecated property, but some clients like Yahoo doesn't support the background property properly and background property acts as a fallback`

Reply:
Well I don't think those comment are mandatory to understand the code, most of the hack we use in MJML are shown either in litmus forum or in this repo https://github.com/hteumeuleu/email-bugs/issues